exoctk/contam_visibility/visibilityPA.py

```diff
@@ -259,7 +259,7 @@ def using_gtvt(ra, dec, instrument, ephFileName=None, output='bokeh'):
         The plotted figure.
 
     """
-
+    print('FOO', ra, type(ra), dec, type(dec))

```

Comment:
is this print statement meant to be here?

Reply:
Haha, no.